addon/mixins/admin-model-mixin.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember'
+import HasManyQuery from 'ember-data-has-many-query'
+
+const {
+  Mixin
+} = Ember
+const { ModelMixin } = HasManyQuery
+
+export default Mixin.create(ModelMixin, {
```

Comment:
I like the thinking here.  Curious if you thought about extending the store rather than the model so that the api is more inline with ember data.  

It might be a little irregular as would have to pass the parent model as an arg or something to that effect.

Thoughts?

Reply:
I didn't think about that, no.

How would usage work with that? Do you mean just using the 
```js
user.get('servers').addObject(myServer)
user.save()
```

Ember Data interface?